carbon-design-system/carbon
# feat(styles): add css bundles into package

Closes #11489

This PR adds in a `build` task which generates prebuilt CSS files in the `styles` package.

#### Changelog

**New**

**Changed**

- Update import ordering of `index.scss`
- Add `build.js` task and script in `package.json`

**Removed**

#### Testing / Reviewing

- Pull down the PR
- Run `yarn build` in `packages/styles`
- Verify that `css/styles.css` and `css/styles.min.css` have been generated
- Inspect their contents to make sure that styles are actually being generated 😅 
- Verify that `css` is included in the `files` array of `package.json`
